aws_cloudfront_distribution.tf

```diff
@@ -14,7 +14,7 @@ resource "aws_cloudfront_distribution" "s3-distribution-TLS-v1" {
     }
   }
 
-  enabled             = true
+  enabled = true
 
   default_cache_behavior {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Use secure ciphers in CloudFront distribution**
 
**Vulnerability**
Use secure ciphers in CloudFront distribution
 
**Remediation**
Enforce the use of secure ciphers TLS v1.1 and TLS v1.2 in a CloudFront Distribution certificate configuration. This is a best security practice. This signature scans for any deviations from this practice and returns the results.

source: accurics-